src/ports/catalog/component.ts

```diff
@@ -0,0 +1,74 @@
+import { IPgComponent } from '@well-known-components/pg-component'
+import { Network } from '@dcl/schemas'
+import {
+  CatalogFilters,
+  CatalogItem,
+  ICatalogComponent,
+  CollectionsItemDBResult,
+} from './types'
+import {
+  getSubgraphNameForNetwork,
+  getCatalogQuery,
+  fromCollectionsItemDbResultToCatalogItem,
+} from './utils'
+import {
+  getCollectionsChainId,
+  getMarketplaceChainId,
+} from '../../logic/chainIds'
+import { getLatestSubgraphSchema } from './queries'
+
+export function createCatalogComponent(options: {
+  database: IPgComponent
```

Comment:
WDYT about renaming the property `database` to `satsumaDatabase` or something similar to explicit that we are not using our DB?

Reply:
I think this catalog component can be re-used in the future by another database (let's say our own one) so at this point, it makes sense it's agnostic to who owns it. At the component level, when the database component is created, I called it `satsumaDatabase` https://github.com/decentraland/nft-server/pull/266/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R686. Does it make sense?